test/cmdlineTests/require_with_error_ir/output

```diff
@@ -0,0 +1,201 @@
+            }
+
+            function identity(value) -> ret {
+                ret := value
+            }
+
+            function convert_t_rational_1_by_1_to_t_uint256(value) -> converted {
+                converted := cleanup_t_uint256(identity(cleanup_t_rational_1_by_1(value)))
+            }
+
+            function abi_encode_t_rational_1_by_1_to_t_uint256_fromStack(value, pos) {
+                mstore(pos, convert_t_rational_1_by_1_to_t_uint256(value))
+            }
+
+            function array_storeLengthForEncoding_t_string_memory_ptr_fromStack(pos, length) -> updated_pos {
+                mstore(pos, length)
+                updated_pos := add(pos, 0x20)
+            }
+
+            function store_literal_in_memory_332c39dcd398ea34a48b871898d589f55fc4c7bce00562fb670c972e7e1b0720(memPtr) {
+
+                mstore(add(memPtr, 0), "two")
+
+            }
+
+            function abi_encode_t_stringliteral_332c39dcd398ea34a48b871898d589f55fc4c7bce00562fb670c972e7e1b0720_to_t_string_memory_ptr_fromStack(pos) -> end {
+                pos := array_storeLengthForEncoding_t_string_memory_ptr_fromStack(pos, 3)
+                store_literal_in_memory_332c39dcd398ea34a48b871898d589f55fc4c7bce00562fb670c972e7e1b0720(pos)
+                end := add(pos, 32)
+            }
+
+            function abi_encode_tuple_t_rational_1_by_1_t_stringliteral_332c39dcd398ea34a48b871898d589f55fc4c7bce00562fb670c972e7e1b0720__to_t_uint256_t_string_memory_ptr__fromStack(headStart , value0) -> tail {
+                tail := add(headStart, 64)
+
+                abi_encode_t_rational_1_by_1_to_t_uint256_fromStack(value0,  add(headStart, 0))
+
+                mstore(add(headStart, 32), sub(tail, headStart))
+                tail := abi_encode_t_stringliteral_332c39dcd398ea34a48b871898d589f55fc4c7bce00562fb670c972e7e1b0720_to_t_string_memory_ptr_fromStack( tail)
+
+            }
+
+            function require_helper_error(condition , expr_15) {
+                if iszero(condition) {
+                    let memPtr := allocate_unbounded()
+                    mstore(memPtr, 0x97ea5a2f00000000000000000000000000000000000000000000000000000000)
+                    let end := abi_encode_tuple_t_rational_1_by_1_t_stringliteral_332c39dcd398ea34a48b871898d589f55fc4c7bce00562fb670c972e7e1b0720__to_t_uint256_t_string_memory_ptr__fromStack(add(memPtr, 4) , expr_15)
+                    revert(memPtr, sub(end, memPtr))
+                }
+            }
+
+            /// @ast-id 21
+            /// @src 0:114:217  "function f(bool condition) external pure..."
+            function fun_f_21(var_condition_9) {
+
+                /// @src 0:177:186  "condition"
+                let _1 := var_condition_9
+                let expr_13 := _1
+                /// @src 0:200:201  "1"
+                let expr_15 := 0x01
+                /// @src 0:169:210  "require(condition, CustomError(1, \"two\"))"
+                require_helper_error(expr_13, expr_15)
```

Comment:
@ekpyron why is this function name generated without a hash in the signature? Pretty sure it should have one.

Reply:
I'll have a look - but this means you should definitely also have semantic tests with two distinct errors with the same arguments, but different names :-).